tests/flow/test_bruteforce.py

```diff
@@ -652,7 +652,7 @@ def test_batch_iterator(self):
     def range_query(self, dist_func):
         bfindex = self.create_index(VecSimMetric_Cosine)
         label_to_vec_list = self.create_add_vectors(bfindex)
-        radius = 0.7
+        radius = bfindex.knn_query(self.query_data[0], k=100)[1][0][-1] # get the distance of the 100th closest vector as the radius
```

Comment:
why?

Reply:
because the radius is related to the data type and data set, and fixing it to a constant value makes no sense (and it also failed as with `UINT8` we found no results).
This way, we get a radius we know matches at least 100 results, regardless of the vectors and their type

Reply:
For better readability, consider splitting this into steps, something like
k100_dists = bfindex.knn_query(self.query_data[0], k=100)[1][0]
last_dist = k100_dists[-1]

you can do better with the names

Reply:
I don't think it makes it more readable